docs/PacketFence_Installation_Guide.asciidoc

```diff
@@ -4535,6 +4535,207 @@ Now go in `Nodes` and click on a MAC address and you will see that a new tab app
 image::TAB.png[scaledwidth="100%",alt="Node TAB"]
 
 
+Monit
+~~~~~
+
+`monit` is a utility for managing and monitoring processes, files, directories and filesystems on a Unix system. Monit conducts automatic maintenance and repair and can execute meaningful causal actions in error situations. E.g. Monit can start a process if it does not run, restart a process if it does not respond and stop a process if it uses too much resources.
+
+Install Monit
+^^^^^^^^^^^^^
+
+The following must be done on each server of a cluster
+
+RHEL / CentOS
+----
+yum localinstall http://packetfence.org/downloads/PacketFence/RHEL6/`uname -i`/RPMS/packetfence-release-1.2-5.1.noarch.rpm -y
+yum install monit --enablerepo=packetfence-extra -y
+----
+
+Debian
+----
+apt-get update
+apt-get install monit
+----
+
+
+Fetch the script signing key
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+----
+gpg --keyserver keyserver.ubuntu.com --recv E3A28334
+----
+
+If the above fails:
+----
+gpg --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys E3A28334
+----
+
+
+Download the Packetfence monit addons
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+----
+cd /tmp/ && wget http://inverse.ca/downloads/PacketFence/monitoring-scripts/v1/monit.tgz && tar xzvf monit.tgz
+----
+
+
+Replace the monit addon directory
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+----
+mv /usr/local/pf/addons/monit /usr/local/pf/addons/monit.old ; mv /tmp/monit /usr/local/pf/addons/
+----
+
+
+Generate the monit configuration
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+----
+/usr/local/pf/addons/monit/monit_build_configuration.pl 'email(s)' 'subject' 'configurations' 'mailserver'
+----
+
+Where :
+[options="compact"]
+* `email(s)`: CSV (no spaces) of recipient email addresses to send alerts.
+* `subject`: Email subject line (example: 'Packetfence Monit Alert').
```

Comment:
Its not the subject, its actually more just a prefix we put to identify customers. Maybe we could just explain it as a prefix but I don't have a good idea for an explanation so it could just stay as-is